app/controllers/project_database.py

```diff
@@ -847,3 +847,38 @@ def get(self, database_id):
             return dict(status='fail', message=errors), 500
 
         return dict(status='success', data=json.loads(database_password_data)), 200
+
+
+class DatabaseStatsView(Resource):
+    @admin_required
+    def get(self):
+        """
+        """
+        database_schema = ProjectDatabaseSchema(many=True)
+        
+        databases = ProjectDatabase.find_all()
+        print(databases)
+
+        validated_database_data, errors = database_schema.dumps(databases)
+
+        if errors:
+            return dict(status='fail', message='Internal Server Error'), 500
+        
+        # get total number of databases
+        databases_data_list = json.loads(validated_database_data)
+        database_count = len(databases_data_list)
+
+        # get databases count per flavour
+
+        flavour_list = []
+        for database in databases_data_list:
```

Comment:
Hey @MutegekiHenry, for more efficiency, instead of using two loops why don't you send another query to get databases `where flavour_name`, Just wondering

Reply:
I didn't want to dictate the flavour names in the code, neither did I want the flavour names to be sent in the request object.

Reply:
We already know flavour names. thing is, for load balancing issues, this takes up more processing than just a query, and i feel it's not necessary

Reply:
what happens when flavours are added or removed, it is what I wanted to cater for.

Reply:
I wanted to avoid the manual handling of flavours

Reply:
Let me rethink the design to make it more effecient.